module/VuFindTheme/src/VuFindTheme/AbstractCssPreCompiler.php

```diff
@@ -187,19 +187,19 @@ protected function getAllThemes()
      * Convert fake absolute paths to working relative paths.
      *
      * @param string $css  Generated CSS
-     * @param string $less Relative LESS filename
+     * @param string $scss Relative SCSS filename
      *
      * @return string
      *
      * @SuppressWarnings(PHPMD.UnusedLocalVariable)
      */
-    protected function makeRelative($css, $less)
+    protected function makeRelative($css, $scss)
```

Comment:
As far as I can tell, this method is never used. Should we just remove it, or am I missing something?

Reply:
The whole PHP-based SCSS compilation command is obsolete and doesn't work. But I'd rather remove it in a separate commit.